2199/3/app/comment_analyzer.rb

```diff
@@ -0,0 +1,52 @@
+require 'dotenv/load'
+# analyze comment
+class CommentAnalyzer
+  ACCESS_KEY = ENV['ACCESS_KEY']
+  AZURE_ENDPOINT = ENV['AZURE_ENDPOINT']
```

Comment:
Does Azure endpoint contain responsive information?
If no, why should we store it in `.ENV` variable?